.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,32 @@
+name: Sphinx
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - name: Install Datasketches and Sphinx
+        run: python -m pip install . sphinx==7.2.4 sphinx-rtd-theme
```

Comment:
with luck we'll be able to remove the pinned version, or else use the fix in the issue about this: https://github.com/sphinx-doc/sphinx/issues/11662#issuecomment-1711017608